lib/msal-browser/src/cache/ITokenCache.ts

```diff
@@ -3,12 +3,16 @@
  * Licensed under the MIT License.
  */
 
-import { ExternalTokenResponse } from "@azure/msal-common";
+import { ExternalTokenResponse, CacheRecord } from "@azure/msal-common";
 import { SilentRequest } from "../request/SilentRequest";
 import { LoadTokenOptions } from "./TokenCache";
 
 export interface ITokenCache {
 
-    /** API to side-load tokens to MSAL cache */
-    loadExternalTokens(request: SilentRequest, response: ExternalTokenResponse, options: LoadTokenOptions): void;
+    /**
+     * API to side-load tokens to MSAL cache
+     * @returns A `CacheRecord` containing the entities that were loaded.
+     */
+    loadExternalTokens(request: SilentRequest, response: ExternalTokenResponse, options: LoadTokenOptions): CacheRecord;
```

Comment:
This makes `CacheRecord` and all of the "Entities" a part of our public API surface, which they were not intended to be. Which particular fields do you need to be returned? I'd be inclined to create a new type which contains what you need rather than exposing internal objects.

Reply:
Perhaps we should use the `AuthenticationResult` type that all of our other APIs return?

Reply:
For our purposes we want to know which account was used and is encoded in the ExternalTokenResponse, so something like the `AccountInfo` would be enough for us.

Reply:
AuthenticationResult (with only relevant fields populated) would work.

Reply:
Okay I can try that. It looks like the `ResponseHandler.generateAuthenticationResult` helper should populate the correct fields?

Reply:
Updated. I wasn't able to use `ResponseHandler.generateAuthenticationResult`, but I copied the relevant bits for TokenCache to do the same logic.